src/nft/components/bag/BagFooter.tsx

```diff
@@ -375,7 +378,7 @@ export const BagFooter = ({ totalEthPrice, fetchAssets, eventProperties }: BagFo
     handleClick,
     buttonColor,
   } = useMemo(() => {
-    let handleClick = fetchAssets
+    let handleClick: (() => void) | (() => Promise<void>) = fetchAssets
```

Comment:
you should just be able to do `() => void`

Reply:
tsserver does not like that